redash/monitor.py

```diff
@@ -36,4 +36,14 @@ def get_status():
         'data_sources': ''
     }
 
+    status['database_metrics'] = []
+    # have to include the fake FROM in the SQL to prevent an IndexError
+    queries = [
+        ['Query Results Size', "pg_size_pretty(pg_total_relation_size('query_results')) as size from (select 1) as a"],
+        ['Redash DB Size', "pg_size_pretty(pg_database_size('postgres')) as size from (select 1) as a"]
+    ]
+    for query_name, query in queries:
+        result = models.db.session.query(query).first()
```

Comment:
You can use `models.db.session.execute` to avoid the weird query, i.e.:

```
models.db.session.execute("SELECT pg_size_pretty(pg_total_relation_size('query_results'))")
```

Reply:
I tried this but ran into errors regarding the api jsonify() call. Example error: `TypeError: <sqlalchemy.engine.result.ResultProxy object at 0x7f8f9e1e6510> is not JSON serializable` related to this: 
```
server_1    |   File "/app/redash/handlers/__init__.py", line 20, in status_api
server_1    |     return jsonify(status)
```

Reply:
Are you sure you were serializing the right thing?

`models.db.session.execute("SELECT pg_size_pretty(pg_total_relation_size('query_results'))")` returns a `ResultProxy` just like `models.db.session.query(query)` does. Make sure you call `.first()` on it to get the first row.

This will also remove the need to make changes in `redash.metrics.database`.

Reply:
Ah. I got too literal in interpreting your first comment. Thanks.